src/server/active_tran_server.cpp

```diff
@@ -241,8 +248,7 @@ active_tran_server::connect_to_page_server (const cubcomm::node &node, const cha
   er_log_debug (ARG_FILE_LINE, "Transaction server successfully connected to the page server. Channel id: %s.\n",
 		srv_chn.get_channel_id ().c_str ());
 
-  assert (m_page_server_conn == nullptr);
```

Comment:
this assert was here for the purpose to prevent all this code from being called twice (albeit buried in the implementation); the purpose of this assert should be served by another check somewhere more to the beginning of the connections loop